packages/hash/api/src/graphql/resolvers/ontology/data-type.ts

```diff
@@ -1,51 +1,69 @@
 import { ApolloError } from "apollo-server-express";
 import { AxiosError } from "axios";
 
-import {
-  PersistedDataType,
-  QueryGetDataTypeArgs,
-  ResolverFn,
-} from "../../apiTypes.gen";
+import { Subgraph, QueryGetDataTypeArgs, ResolverFn } from "../../apiTypes.gen";
 import { GraphQLContext, LoggedInGraphQLContext } from "../../context";
-import { DataTypeModel } from "../../../model";
-import { mapDataTypeModelToGQL } from "./model-mapping";
 
 export const getAllLatestDataTypes: ResolverFn<
-  Promise<PersistedDataType[]>,
+  Promise<Subgraph>,
   {},
   LoggedInGraphQLContext,
   {}
 > = async (_, __, { dataSources }) => {
   const { graphApi } = dataSources;
 
-  const allLatestDataTypeModels = await DataTypeModel.getAllLatest(
-    graphApi,
-  ).catch((err: AxiosError) => {
-    throw new ApolloError(
-      `Unable to retrieve all latest data types. ${err.response?.data}`,
-      "GET_ALL_ERROR",
-    );
-  });
+  const { data: dataTypeSubgraph } = await graphApi
+    .getDataTypesByQuery({
+      query: { eq: [{ path: ["version"] }, { literal: "latest" }] },
+      /** todo - make these configurable once non-primitive data types are a thing https://app.asana.com/0/1200211978612931/1202464168422955/f */
```

Comment:
```suggestion
      /** @todo - make these configurable once non-primitive data types are a thing https://app.asana.com/0/1200211978612931/1202464168422955/f */
```

Reply:
Oops not sure how I missed these, thank you